package.json

```diff
@@ -4,23 +4,24 @@
   "description": "Client-side library to interact with the Tidepool platform",
   "main": "tidepool.js",
   "scripts": {
-    "test": "grunt test"
+    "test": "grunt test",
+    "security-checks": "npm audit --production --registry=https://registry.npmjs.org/"
   },
   "repository": "https://github.com/tidepool-org/platform-client.git",
   "author": "Tidepool",
   "license": "BSD-2-Clause",
   "dependencies": {
-    "async": "3.1.0",
+    "async": "3.2.0",
     "lodash": "4.17.15",
-    "superagent": "5.1.0",
-    "uuid": "3.3.3"
+    "superagent": "5.2.2",
+    "uuid": "8.1.0"
   },
   "devDependencies": {
-    "grunt": "1.0.4",
+    "chai": "4.2.0",
+    "grunt": "1.1.0",
     "grunt-cli": "1.3.2",
     "grunt-contrib-jshint": "2.1.0",
     "grunt-mocha-test": "0.13.3",
-    "mocha": "6.2.2",
-    "salinity": "0.0.8"
+    "mocha": "8.0.1"
```

Comment:
general comment: why do we make those upgrade in this PR ?
do we need them to address the PR objective ?

Reply:
They embark dependencies with vulnerabilities.
which then comes to blip, replacing sometimes fixed packages versions.
npm seems to use the oldest version on conflict (at least I got a problem with one dep).